src/product/infra/persistence/mapper/product.mapper.ts

```
@@ -0,0 +1,48 @@
+    entity.unit = domain.unit;
+    entity.capacity = domain.capacity;
+    entity.specification = domain.specification;
+    entity.description = domain.description;
+    entity.isRecovarable = domain.isRecovarable;
+    entity.status = domain.status;
+    entity.categories = domain.categories.map((category) =>
+      CategoryMapper.toEntity(category),
+    );
+    return entity;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)